docs/encyclopedia/nexus-operations.mdx

```diff
@@ -0,0 +1,184 @@
+  - Start a Workflow as an asynchronous Operation.
+- New-Sync-Operation
+  - Invoke an underlying Query or Signal as a synchronous Operation.
+  - Invoke an Update as a synchronous Operation.
+  - Execute arbitrary code as a synchronous Operation.
+
+Nexus Operations are not intrinsically durable, but can be made durable by backing them with Temporal primitives like a Workflow, Update, or Signal.
+Operations may be implemented with Temporal primitives or execute arbitrary code, for example to make a database query or call a microservice.
+
+## Nexus Operation Lifecycle {#operation-lifecycle}
+
+When you execute a Nexus Operation from a caller Workflow using the Temporal SDK, a command is sent to Temporal to schedule the Nexus Operation which is atomically handed off to the Nexus Machinery.
+The [Nexus Machinery](/glossary#nexus-machinery) uses state-machine-based invocation and completion callbacks to ensure [at-least-once](#at-least-once-execution-semantics-and-idempotency) execution of a Nexus Operation and reliable delivery of the result.
+
+![Nexus Overview](/img/nexus-overview.png)
+
+The Nexus Machinery is responsible for making the Nexus RPC calls on behalf of the caller Workflow, with [automatic-retries](#automatic-retries).
+This means you don’t have to use Nexus RPC directly, only the Temporal SDK along with the Temporal Service.
+
+### System interactions
+
+Nexus Operations are often registered in the same Worker as the underlying Temporal primitives they abstract.
+
+![NexusYourCloud](/img/nexus/nexus-connect-queue-based-workers.png)
+
+For example, when you execute a Nexus Operation in a caller Workflow:
+1. **RespondWorkflowTaskCompleted([ScheduleNexusOperation command](/references/commands#schedulenexusoperation))** \-
+is used by the caller Worker to schedule a Nexus Operation, which atomically hands off execution to the caller's Nexus Machinery.
+2. **PollNexusTaskQueue** \-
+is used by the handler Worker to recieve a [Nexus Task](/workers#nexus-task) to process, for example to start a Nexus Operation.
+3. **RespondNexusTaskCompleted** or **RespondNexusTaskFailed** \-
+is used by the handler Worker to return the Nexus Task result.
+When asked to start a Nexus Operation, the Nexus handler decides if the Operation will be synchronous or asynchronous.
+   1. This is typically a static decision based on the [Temporal SDK builder function used](#sdk-support).
+   2. [Asynchronous Nexus Operations](#asynchronous-operation-lifecycle), created with the New-Workflow-Run-Operation SDK helper, will return a Nexus Operation ID, that can be used to perform additional actions like canceling an Operation.
+   3. [Synchronous Nexus Operations](#synchronous-operation-lifecycle), created with the New-Sync-Operation SDK helper, will return the Operation result directly.
+The caller's Nexus Machinery recieves the result and records a [NexusOperation event](/references/events#nexusoperationstarted) in the caller's Workflow History.
```

Comment:
I would clarify that this isn't necessarily a NexusOperationStarted event, this could be any of the Nexus "progress" events and that there's no such thing as a `NexusOperation` event.

Reply:
Fair point. Was trying to keep it generic with `[NexusOperation event]` and linking to the first event in the reference docs, but will make this more explicit.

Reply:
fixed